src/katgpucbf/fgpu/send.py

```diff
@@ -150,17 +172,24 @@ def _inc_counters(frame: Frame, future: asyncio.Future) -> None:
                 output_clip_counter.labels(pol).inc(frame.saturated[pol])
 
     async def send(
-        self, stream: "spead2.send.asyncio.AsyncStream", frames: int, time_converter: TimeConverter, sensors: SensorSet
+        self,
+        streams: list["spead2.send.asyncio.AsyncStream"],
+        frames: int,
+        time_converter: TimeConverter,
+        sensors: SensorSet,
     ) -> None:
-        """Transmit heaps on a SPEAD stream.
+        """Transmit heaps over SPEAD streams.
 
-        Frames from 0 to `frames` - 1 are sent asynchronously.
+        Frames from 0 to `frames` - 1 are sent asynchronously. The contents of
+        each frames are distributed over the streams. If the number of streams
+        does not divide into the number of destination endpoints, there will be
+        imbalances, because the partitioning is the same for every frame.
```

Comment:
i.e. so it'll work, but you may not get the performance you expect?

Reply:
If the imbalance is more than a few percent I think it won't work, because each of the N streams is rate-limited to 1/N of the total rate (plus a safety margin). So if there is an imbalance then the most-loaded stream will get heaps in faster than it's allowed to send them out. main.py has a safety check to prevent this happening.